packages/core/data-transfer/lib/providers/local-file-destination-provider/index.ts

```diff
@@ -96,6 +95,10 @@ class LocalFileDestinationProvider implements IDestinationProvider {
     return transforms;
   }
 
+  createGzip(): zlib.Gzip {
```

Comment:
This should be private if it's here, but why is it better than calling zlib.createGzip as needed?

Reply:
Yes, but this was the only way to test it because the zlib library doesn't work well with typescript and jest. Basically, it throws an error that is a read-only property when you try to spy on a method... 😢